device/jsonld-contexts/device.jsonld

```diff
@@ -1,14 +1,21 @@
 {
     "@context": {
         "Device": "https://smartdatamodels.org/dataModel.Device/Device",
+        "appEUI": "https://smartdatamodels.org/dataModel.Device/appEUI",
         "applicationId": "https://smartdatamodels.org/dataModel.Device/applicationId",
         "batteryVoltage": "https://smartdatamodels.org/dataModel.Device/batteryVoltage",
         "chargerStatus": "https://smartdatamodels.org/dataModel.Device/chargerStatus",
+        "devEUI": "https://smartdatamodels.org/dataModel.Device/devEUI",
+        "deviceDescription": "https://smartdatamodels.org/dataModel.Device/deviceDescription",
+        "deviceIsDeleted": "https://smartdatamodels.org/dataModel.Device/isDeleted",
         "deviceName": "https://smartdatamodels.org/dataModel.Device/deviceName",
+        "deviceNetworks": "https://smartdatamodels.org/dataModel.Device/deviceNetworks",
         "deviceType": "https://smartdatamodels.org/dataModel.Device/deviceType",

```

Comment:
Personally I just followed the nomenclature of deviceName and deviceType already present.

Reply:
I used deviceType for obvious reasons but it is true that deviceName may not be necessary ...
When seeing it written on all the attributes I agree that we should just keep name, description, isDeleted.
Same for networkName, it can just be name (since it is a subproperty of networks it is clear enough that it is the name of the network in my opinion).
Also networkIsEnabled could just be isEnbaled too, or deviceIsEnabled maybe (it is not the network that is enabled but the device that is enabled on the network so it is a bit misleading)